templates/thank-you.html

```diff
@@ -16,20 +15,62 @@ <h1 data-qa="title">{{ _("Submission successful") }}</h1>
     "classes": "u-mb-s"
     })
   %}
+    <h1 data-qa="title">{{ _("Thank you for submitting the {survey_title}").format(
+      survey_title = survey_title)}}</h1>
     <p data-qa="guidance">
-      {% if content.trad_as %}
-        {{ _("Your answers were submitted for <span>{ru_name}</span> ({trading_as_name}) on {submitted_date_time}").format(
+      {% if survey_type == "social" %}
+        {{ _("Your answers have been submitted") }}
+      {% elif content.trad_as %}
+        {{ _("Your answers have been submitted for <span>{ru_name}</span> ({trading_as_name})").format(
           ru_name = content.ru_name,
-          trading_as_name = content.trad_as,
-          submitted_date_time=content.submitted_at|format_datetime) }}
+          trading_as_name = content.trad_as) }}
       {% else %}
-        {{ _("Your answers were submitted for <span>{ru_name}</span> on {submitted_date_time}").format(
-          ru_name = content.ru_name,
-          submitted_date_time=content.submitted_at|format_datetime) }}
+        {{ _("Your answers have been submitted for <span>{ru_name}</span>").format(
+          ru_name = content.ru_name) }}
       {% endif %}
     </p>
-
-    <p>{{ _("Transaction ID: <b>{transaction_id}</b>").format(transaction_id=content.tx_id)}}</p>
+    {% if survey_type == "social" %}
+      {{ onsMetadata({
+        "termCol": 3,
```

Comment:
When I tried a social survey it looked like this
![image](https://user-images.githubusercontent.com/14179817/129343660-510c1ee3-762f-4cc0-9738-b8bbfb06ed4c.png)